src/content/docs/fr/reference/configuration-reference.mdx

```diff
@@ -0,0 +1,1711 @@
+
+Les paramètres régionaux par défaut de votre site web/application. Ceci est un champ obligatoire.
+
+Aucun format de langue ou syntaxe particulière n'est imposé, mais nous vous suggérons d'utiliser des minuscules et des traits d'union si nécessaire (par exemple « es », « pt-br ») pour une meilleure compatibilité.
+
+### i18n.locales
+
+<p>
+
+**Type :** `Locales`<br />
+<Since v="3.5.0" />
+</p>
+
+Une liste de tous les paramètres régionaux pris en charge par le site Web, y compris `defaultLocale`. Ceci est un champ obligatoire.
+
+Les langues peuvent être répertoriées soit sous forme de codes individuels (par exemple `['en', 'es', 'pt-br']`) soit en associant un chemin (`path`) à un tableau de codes (par exemple `{ path: "english", codes: ["en", "en-US"]}`). Ces codes seront utilisés pour déterminer la structure de l'URL de votre site déployé.
+
+Aucun format particulier de code de langue ou de syntaxe n'est appliqué, mais les dossiers de votre projet contenant vos fichiers de contenu doivent correspondre exactement aux éléments de la liste `locales`. Dans le cas de plusieurs `codes` pointant vers un préfixe de chemin d'URL personnalisé, stockez vos fichiers de contenu dans un dossier portant le même nom que le chemin configuré (`path`).
+
+### i18n.fallback
+
+<p>
+
+**Type :** `Record.<string, string>`<br />
+<Since v="3.5.0" />
+</p>
+
+La stratégie de repli lors de la navigation vers des pages qui n'existent pas (par exemple une page traduite n'a pas été créée).
+
+Utilisez cet objet pour déclarer une route de repli pour chaque langue que vous prenez en charge. Si aucune solution de repli n'est spécifiée, les pages indisponibles renverront un code d'état 404.
+
+##### Exemple
+
+L'exemple suivant configure votre stratégie de repli pour rediriger les pages indisponibles dans `/pt-br/` vers leur version `es`, et les pages indisponibles dans `/fr/` vers leur version `en`. Les pages `/es/` non disponibles renverront un code d'état 404.
+
+```js
+export default defineConfig({
+	i18n: {
+		defaultLocale: "en",
+		locales: ["en", "fr", "pt-br", "es"],
+		fallback: {
+			pt: "es",
+		  fr: "en"
+		}
+	}
+})
+```
+
+### i18n.routing
+
+<p>
+
+**Type :** `Routing`<br />
+<Since v="3.7.0" />
+</p>
+
+Contrôle la stratégie de routage pour déterminer les URL de votre site. Définissez ceci en fonction de la configuration de votre chemin de dossier/URL pour votre langue par défaut.
+
+#### i18n.routing.prefixDefaultLocale
+
+<p>
+
+**Type :** `boolean`<br />
+**Par défaut :** `false`<br />
+<Since v="3.7.0" />
+</p>
+
+Lorsque le paramètre est défini sur `false`, seules les langues autres que celles par défaut afficheront un préfixe de langue.
+La langue par défaut (`defaultLocale`) n'affichera pas de préfixe de langue et les fichiers de contenu ne résident pas dans un dossier localisé.
+Les URL seront de la forme `exemple.com/[locale]/content/` pour toutes les langues autres que celles par défaut, mais `exemple.com/content/` pour la langue par défaut.
```

Comment:
```suggestion
Les URL seront de la forme `example.com/[locale]/content/` pour toutes les langues autres que celles par défaut, mais `example.com/content/` pour la langue par défaut.
```

Reply:
Not sure why it is outdated but after looking for `exemple.com` in the file it seems to be solved!